zigbee2mqtt/config.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "Zigbee2mqtt",
-    "version": "0.1",
+    "version": "latest",
```

Comment:
This is a hack because it uses the version to fetch the image from docker hub based on a tag matching this version. 
With `latest` the image is pulled, but there is a stacktrace on start:
```
18-05-10 17:50:25 ERROR (MainThread) [aiohttp.server] Error handling request
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/aiohttp/web_protocol.py", line 381, in start
    resp = await self._request_handler(request)
  File "/usr/lib/python3.6/site-packages/aiohttp/web_app.py", line 322, in _handle
    resp = await handler(request)
  File "/usr/lib/python3.6/site-packages/aiohttp/web_middlewares.py", line 88, in impl
    return await handler(request)
  File "/usr/lib/python3.6/site-packages/hassio/api/security.py", line 48, in token_validation
    return await handler(request)
  File "/usr/lib/python3.6/site-packages/hassio/api/utils.py", line 33, in wrap_api
    answer = await method(api, *args, **kwargs)
  File "/usr/lib/python3.6/site-packages/hassio/addons/utils.py", line 33, in wrap_check
    return await method(addon, *args, **kwargs)
  File "/usr/lib/python3.6/site-packages/hassio/addons/addon.py", line 652, in start
    return await self.instance.run()
  File "/usr/lib/python3.6/site-packages/hassio/utils/__init__.py", line 25, in wrap_api
    return await method(api, *args, **kwargs)
  File "/usr/lib/python3.6/concurrent/futures/thread.py", line 56, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/lib/python3.6/site-packages/hassio/docker/addon.py", line 263, in _run
    self.image, self.version)
  File "/usr/lib/python3.6/site-packages/hassio/docker/addon.py", line 47, in version
    return super().version
  File "/usr/lib/python3.6/site-packages/hassio/docker/interface.py", line 45, in version
    if self._meta and LABEL_VERSION in self._meta['Config']['Labels']:
TypeError: argument of type 'NoneType' is not iterable
``